incubator/chartmuseum/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
 name: chartmuseum
-version: 0.1.0
+version: 0.2.7
```

Comment:
Quick clarification. Any reason for the bump of the chart version to 0.2.7? Perhaps just a minor upgrade of the chart version would suffice here (i.e. 0.1.0 => 0.1.1)?

Reply:
I thought the minor was appropriate since it was not a very small change. See this PR for versions and a note about the lint failures  https://github.com/kubernetes/charts/pull/2746 

With that said I did not update values.yaml..

Reply:
I realized I misinterpreted the request. Pushed the fix for versions and bumped docker image per other PR.